Gemfile

```diff
@@ -8,4 +8,5 @@ gem "rspec"
 gem "webmock"
 gem "colorize"
 gem "google_drive", "~> 2.1"
-gem "gds-api-adapters"
+gem 'gds-api-adapters', '~> 49.1'
```

Comment:
What's bump needed for?

Reply:
This change isn't needed (I'll revert it). Originally I couldn't get the `GdsApi::PublishingApiV2` class working, so I naively bumped the version thinking it would fix my problems. The real problem was because I didn't [`require` the adapter](https://github.com/alphagov/govuk-tagging-monitor/pull/31/files#diff-b2451cb79a43f2924e5505a79f4a23b8R5) 😇